src/Exceptions/InvalidDumperException.php

```diff
@@ -0,0 +1,24 @@
+<?php namespace Geocoder\Laravel\Exceptions;
```

Comment:
(All code is valid PSR1 and PSR2.) I have adopted moving the namespace to the same line as the PHP tag to save space. For me it makes contextual sense to have them on the same line. If this is something you don't like, I will move namespace declarations to their own line. :)